Packs/CortexCloud/Playbooks/Cortex_Cloud_-_AWS_EC2_Instance_Misconfigurations_Remediation.yml

```diff
@@ -0,0 +1,730 @@
+id: 62374b53-6e1b-4c9c-8ba8-0ec3889d64a1
```

Comment:
The `id` field must match the `name` field, and the `version` field should be -1.  These can be corrected automatically using the `validate` function in demisto-sdk.

Reply:
@shnpanw I see the `version` field has been updated.  Can you also update `id` to match the value you nave in `name` please?